cmd/options.go

```diff
@@ -148,6 +151,7 @@ func (c *compactOptions) addFlags(fs *flag.FlagSet) {
 	c.restorationConfig.AddFlags(fs)
 	c.snapstoreConfig.AddFlags(fs)
 	fs.BoolVar(&c.needDefragmentation, "defragment", c.needDefragmentation, "defragment after compaction")
+	fs.DurationVar(&c.snapshotTimeout.Duration, "compact-snapshot-timeout", c.snapshotTimeout.Duration, "context timeout for taking compacted full snapshots")
```

Comment:
`context timeout for taking compacted full snapshots` -> `timeout duration for taking compacted full snapshots`